src/content/en/billions.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: shared/root
+published: false
+description: "Building Web Apps for the Next Billion Users"
+title: "Building Web Apps for the Next Billion Users"
+translation_priority: 0
+---
+
+<style>
+.mdl-layout__content {
+    padding: 20px 0 0 100px;
+    }
+</style>
+
+<p class="intro">The next billion people coming online will be more diverse than ever, in every respect: physical location, cultural experience, device types, computer expertise and access to connectivity.
```

Comment:
Remove the comma after 'than ever'.

End of sentence: 'and connectivity' instead of 'and access to connectivity'.